CONTRIBUTING.md

```diff
@@ -37,193 +37,210 @@ To add a new page to the website, [create a new markdown file with YAML frontmat
-# Default Value is <abbr title='Long Term Support'>LTS</abbr>
+# The label that will be used alongside releases labelled with `lts: true` (optional, default = "<abbr title='Long Term Support'>LTS</abbr>").
+# Only provide if the product has LTS releases that are not called LTS, but something else.
 # Prefer using an HTML abbr tag, if possible.
 LTSLabel: "<abbr title='Extra Long Support'>ELS</abbr>"
 
-# Optional information about how release information can be fetched automatically
-# This is used for automatically updating `releaseDate`, `latest`, and `latestReleaseDate` for
-# every release.
-# Please see https://github.com/endoflife-date/endoflife.date/wiki/Automation for more details
+# The name of the End of Life column (optional, default = Security Support).
+eolColumn: Service Status
+
+# Whether to display the "Active Support" column (optional, default = false).
+# You can also set this variable with a text label if you want to change the column name:
+#   activeSupportColumn: Customer Support
+activeSupportColumn: false
+
+# Whether to display the "Latest" column (optional, default = true).
+# If the product doesn't have patch releases, set this to false.
+releaseColumn: true
+
+# Whether to show the "Released" column (optional, default = false).
+releaseDateColumn: true
+
+# Whether to show the discontinued column (optional, default = false).
+# Set to true if you're tracking a device.
+# This usually means the device is no longer available for sale or is no longer being manufactured.
+discontinuedColumn: false
+
+# Auto-update release configuration (optional).
+# This is used for automatically updating `releaseDate`, `latest`, and `latestReleaseDate` for every release.
+# Multiple configuration are allowed. Please see https://github.com/endoflife-date/endoflife.date/wiki/Automation for more details.
+# The presence of such configuration enables a footer note on the product page informing users that latest releases are automatically updated.
 auto:
-  # Any valid git clone URL will work
-  # Support for partialClone is necessary (GitHub does support this)
+  # Configuration for auto-update based on git.
+  # Any valid git clone URL will work, but support for partialClone is necessary (GitHub and GitLab support it).
   - git: https://github.com/abc/def.git
-    # An optional regex that defines how the tags above should translate to releases
-    # Use named capturing groups
-    # Default value should work for most releases of the form a.b or a.b.c
-    # default also skips over any special releases (nightly,beta,pre,rc etc)
-    # The default values can be found here: https://github.com/endoflife-date/release-data/blob/main/update.rb#L19-L20
-    # This needs to be a Ruby-Compatible regex
+    # Ruby-compatible regex that defines how the tags above should translate to releases (optional, default can be found on https://github.com/endoflife-date/release-data/blob/main/update.rb#L19-L20).
```

Comment:
this looks too long for a single line , can we divide this line into 2 lines ? ( maybe just  after releases  part and move (optional part to second line

Reply:
and also a space just before  `)` to make url correct

Reply:
Splitted.